app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -130,6 +131,19 @@ public function panel(Panel $panel): Panel
             ])
             ->authMiddleware([
                 Authenticate::class,
+            ])
+            ->plugins([
+                BreezyCore::make()
+                    // See https://github.com/jeffgreco13/filament-breezy
+                    ->myProfile(
+                        shouldRegisterUserMenu: true,
+                        shouldRegisterNavigation: false,
+                        hasAvatars: false,
+                        slug: 'my-profile',
+                    )
+                    ->enableTwoFactorAuthentication(
+                        force: config('filament.enabled'),
```

Comment:
```suggestion
                        force: config('filament.2fa_enabled'),
```

otherwise it could be confusing about it disabling filament/admin backend itself